README.md

```diff
@@ -64,6 +58,18 @@ You can configure the following environment variables:
 Use your preferred Pulumi backend. https://www.pulumi.com/docs/concepts/state/#deciding-on-a-state-backend
 Change the `pulumilocal` command in the instructions above to `pulumi`.
 
+## Change Log
+
+* v1.0: Using `pulumi config set-all` to set all the AWS provider configurating instead of modifying
+* the Stack file directly. Removed defaulting the stack name to `localstack`. Added argparse. 
+* Removed pyyaml dependency. Removed python2 package classifiers. 
```

Comment:
```suggestion
* v1.0: Using `pulumi config set-all` to set all the AWS provider configurating instead of modifying
    the Stack file directly. Removed defaulting the stack name to `localstack`. Added argparse. 
    Removed pyyaml dependency. Removed python2 package classifiers. 
```